core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -199,8 +198,10 @@ class TransactionStateManager(brokerId: Int,
 
   def getTransactionState(transactionalId: String) = getAndMaybeAddTransactionState(transactionalId, None)
 
-  def putTransactionStateIfNotExists(transactionalId: String, txnMetadata: TransactionMetadata) =
```

Comment:
Can you please specify the result type of the public `getTransactionState` method as well?